### PR TITLE
Improve some skin defaults

### PR DESCRIPTION
**Pull request by heff**
learned from creating the new designer. http://codepen.io/heff/pen/EarCt/left/?editors=010

See the commits for a list of changes.

### CODE REVIEW

File: src/css/_private-variables.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-$secondary-bg: lighten($primary-bg, 35%) !default;
+// Control backgrounds (control bar, big play, menus)
+$primary-background-color: #2B333F !default;
+$primary-background-transparency: 0.7 !default;
 
-$text-font-family: Arial, Helvetica, sans-serif !default;
-$base-font-size: 10px !default;
+// Hover states, slider backgrounds
+$lighter-background-color: lighten($primary-background-color, 33%) !default;
+$lighter-background-transparency: 0.5 !default;
```
Comment (mmcc) [inline src/css/_private-variables.scss]: Let's switch this over to `$secondary-background-*`

File: src/css/_private-variables.scss [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-$control-bar--bg: #2B333F !default;
-$control-bar--transparency: 0.5 !default;
+$text-font-family: Arial, Helvetica, sans-serif !default;
 
 $big-play-button--width: 3em !default;
```
Comment (mmcc) [inline src/css/_private-variables.scss]: Just to clear this up after our conversation, add a comment of some kind here solidifying the naming structure for component-specific variables (`$component-name--attribute`).

**Comment by mmcc** [comment]
Minor comments, otherwise lgtm

**Comment by heff** [comment]
Addressed the comments. Pulling in.

**Pull request closed without merging**